scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorJob.java

```diff
@@ -38,6 +39,7 @@ public void onJobAddedToTask(final ActorTask task) {
     schedulingState = TaskSchedulingState.QUEUED;
   }
 
+  @Async.Execute
```

Comment:
:question: How is the job mapped back with the key? i.e. `@Async.Schedule` uses the job as key. How does this know which stack to use? :D

Reply:
It's implicitly "this" which work nicely here :tada: See https://www.jetbrains.com/help/idea/debug-asynchronous-code.html#bc2ffa26

Reply:
Oh, right, `@Async.Execute` is on the `ActorJob` itself. Gotcha. Cool :+1: